server/views/pages/overview/partials/workAndInterestsTab/_personalSkillsAndInterestsSummaryCard.njk

```diff
@@ -1,6 +1,6 @@
 {# Skills and interests summary card #}
-{% set inductionHasSkillsRecorded = induction.inductionDto.inPrisonInterests.skills | length > 0 %}
-{% set inductionHasInterestsRecorded = induction.inductionDto.inPrisonInterests.interests | length > 0 %}
+{% set inductionHasSkillsRecorded = induction.inductionDto.personalSkillsAndInterests.skills | length > 0 %}
```

Comment:
This was the core problem - we were looking at the wrong property previously

I've also added a series of `data-qa` attributes to support finding the correct elements in the unit test 👍